install/installer/pkg/common/resources/latest-migration.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
we could generate this  directly into `service-waiter/cmd` instead of piping it through the installer. I believe that would be simpler and leaves less space for errors

Reply:
Addressed

cc @svenefftinge 🙏